docs/source/performance.rst

```diff
@@ -0,0 +1,41 @@
+.. _pika: https://pika.readthedocs.io/en/stable/
+.. _aio-pika: https://pika.readthedocs.io/en/stable/
+.. _msgpack: https://github.com/msgpack/msgpack-python
+
+
+Performance
+===========
+
+KiwiPy is designed primarily with ease-of-use and robustness in mind with less of a focus on performance.
+Never the less, it is expected that depending on the configuration and network conditions kiwPy should be able to handle hundreds to thousands of messages per second.
+
+The chart below shows benchmarks comparing kiwiPy to `pika`_ using the following configuration:
+
+* Sending 100 task messages consisting of 32 to 2048 bytes in size (default 1024)
+* Messages are sent and _then_ received (i.e. serialised) to be able to show split timings
+* Everything running locally (server and client) on Dell XPS 15 (Core i9-9980H @ 2.3 x 16, 1TB SSD HDD), Ubuntu 20.04
+* Using `msgpack`_ for encoding and decoding of messages
+* Mandatory routing enabled meaning that the code will wait for messages to be routed by the broker (or raise an exception)
+* Full code can be found in ``test/rmq/bench/test_benchmarks.py``
+
+Tests are split into ``send``, ``get`` and ``send_get`` to show a breakdown of the full process of sending and receiving a message.  The ``send_get`` test is carried out for different message size (in bytes) as indicated in square brackets.
+
+
+.. image:: /_static/bench.svg
```

Comment:
For some reason I cannot open this svg on Ubuntu 18.04. Inkscape crashes _hard_. It becomes unresponsive and I have to kiil it with a SIGKILL. Also `eog` cannot open it and reports CSS parsing errors. I built the documentation and can see the image fine in Firefox though. Really weird.

One thing you might want to change is the font of the labels. Even I couldn't read them and that is typically not a good sign for others

Reply:
yeah, strange.  I could load it in inkscape (U20.04) but it didn't quite display correctly.  It could be because there is embedded javascript.

I've upped the size of the labels by a few points.